sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/implementation/ContainerRegistriesImpl.java

```diff
@@ -265,291 +466,2074 @@ Mono<ContainerRegistriesGetManifestsNextResponse> getManifestsNext(
+     * @param contentType The manifest's Content-Type.
+     * @param context The context to associate with this operation.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws AcrErrorsException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return the response.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public ContainerRegistriesCreateManifestResponse createManifestSyncWithResponse(
+            String name,
+            String reference,
+            BinaryData payload,
+            long contentLength,
+            String contentType,
+            Context context) {
+        final String accept = "application/json";
+        return service.createManifestSync(
+                this.client.getUrl(), name, reference, contentType, payload, contentLength, accept, context);
+    }
+
+    /**
+     * Put the manifest identified by `name` and `reference` where `reference` can be a tag or digest.
+     *
+     * @param name Name of the image (including the namespace).
+     * @param reference A tag or a digest, pointing to a specific image.
+     * @param payload Manifest body, can take v1 or v2 values depending on accept header.
+     * @param contentLength The Content-Length header for the request.
+     * @param contentType The manifest's Content-Type.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws AcrErrorsException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void createManifestSync(
+            String name, String reference, BinaryData payload, long contentLength, String contentType) {
+        createManifestSyncWithResponse(name, reference, payload, contentLength, contentType, Context.NONE);
+    }
+
+    /**
+     * Put the manifest identified by `name` and `reference` where `reference` can be a tag or digest.
+     *
+     * @param name Name of the image (including the namespace).
+     * @param reference A tag or a digest, pointing to a specific image.
+     * @param payload Manifest body, can take v1 or v2 values depending on accept header.
+     * @param contentLength The Content-Length header for the request.
+     * @param contentType The manifest's Content-Type.
+     * @param context The context to associate with this operation.
+     * @throws IllegalArgumentException thrown if parameters fail the validation.
+     * @throws AcrErrorsException thrown if the request is rejected by server.
+     * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public void createManifestSync(
```

Comment:
Same as above, should not be generated?
Also true for some of the API's generated below.